Fix issue #226

Karim-Mane opened the pull request:
This PR contains changes to address issue #226. The changes include:

* returning a vector of target columns (instead of a comma-separated strings) for the `duplicates_checked_from` element of the report generated by the `find_duplicates()` or `remove_duplicates()` functions.
* update the NEWS.md file

joshwlambert commented on NEWS.md:
```diff
@@ -7,6 +7,11 @@
+## Enhancements
+
+* The value of the `duplicates_checked_from` element of the report is now a
+vector of target columns (#228, @Karim-Mane).
```
Could be worth mentioning that it used to be a single `character` string to make clear how it being a vector is different to previous versions.

joshwlambert reviewed (approved):
The changes in this PR address #226 for `find_duplicates()` but I think that other exported functions in {cleanepi}, such as `convert_to_numeric()`, still call `toString()` when `add_to_report()` is called in the function. 

Approving this PR as I'll let you decide whether you remove `toString()` in other functions in this PR or a separate one. 

Thanks for quickly addressing the issues! 🚀

Karim-Mane commented:
> The changes in this PR address #226 for `find_duplicates()` but I think that other exported functions in {cleanepi}, such as `convert_to_numeric()`, still call `toString()` when `add_to_report()` is called in the function.
> 
> Approving this PR as I'll let you decide whether you remove `toString()` in other functions in this PR or a separate one.
> 
> Thanks for quickly addressing the issues! 🚀

There were three other instances of this. The conversion into character was introduced to anticipate on how they will be shown in the browser after calling the `print_report()` function. This is now handled internally within the Rmarkdown file used to create the HTML report.
